NAMESPACE

```diff
@@ -24,6 +24,8 @@ export(makeMicrobiomeExperimentFromphyloseq)
 export(meltAssay)
 export(mergeCols)
 export(mergeRows)
+export(prevalence)
```

Comment:
The functions names do not follow conventions. How about `getPrevalence`/`getPrevalenceMembers`?

Reply:
Ok.

Reply:
`Features` or `Taxa` instead of `members` to keep it in sync with `getTopTaxa`?

Reply:
might make sense. @antagomir ?

Reply:
makes sense; naming is tricky - 

`Features` is more generic and can be sometimes more accurate than `Taxa`. But it is also longer and less descriptive. 

Moreover, we might like to use the same functions later for other (more refined) metagenomic units or other multiomics that are available through `ME` (or `TSE`/`MAE` or derivatives). Of course there could be aliases for different cases as well if that's easier.

This naming decision may ultimately have implications for a large number of functions.

Reply:
prevalence is quite specific for microbiome analysis. Thats why I would go for `Taxa`. You can implement `Feature` as a wrapper and be done with it.

Reply:
We might also like check prevalence of some more refined metagenomic features. But this is indeed possible to sort out later shall needs arise.